defaults/plugins/org.zowe.zlux.agent.json

```diff
@@ -0,0 +1,4 @@
+{
+  "identifier": "org.zowe.zlux.agent",
+  "pluginLocation": "../../zlux-server-framework/plugins/zlux-agent"
```

Comment:
this is old and not used style anymore for core plugins because it fails if you switch ROOT_DIR
this is what you need instead
```
  "identifier": "org.zowe.zlux.agent",
  "pluginLocation": "components/app-server/share/zlux-server-framework/plugins/zlux-agent",
  "relativeTo": "$ROOT_DIR"
```

Reply:
updated to migratable location